src/leed_nsym/cleed.c

```
@@ -0,0 +1,267 @@
+static int cleed_option_takes_value(const char *arg)
+{
+  return strcmp(arg, "-b") == 0 ||
+         strcmp(arg, "-i") == 0 ||
+         strcmp(arg, "-o") == 0 ||
+         strcmp(arg, "-r") == 0 ||
+         strcmp(arg, "-w") == 0;
+}
+
+static int cleed_should_show_help(int argc, char *argv[])
+{
+  int i;
```

Clang (14.0.6)

[warning] variable 'i' is not initialized

(cppcoreguidelines-init-variables)